app/clients/sms/firetext.py

```diff
@@ -103,6 +102,6 @@ def send_sms(self, to, content, reference, sender=None, **kwargs):
             raise FiretextClientResponseException(response=e.response, exception=e)
         finally:
             elapsed_time = monotonic() - start_time
-            self.current_app.logger.info("Firetext request for {} finished in {}".format(reference, elapsed_time))
+            self.current_app.logger.info("Firetext request for %s finished in %f", reference, elapsed_time)
```

Comment:
we should be able to use an %s with am if/else condition, any reason why we are not?

Reply:
I don't follow here.  What is the condition?